src/py42/clients/trustedactivities.py

```diff
@@ -0,0 +1,82 @@
+from py42.choices import Choices
+
+
+class TrustedActivityType(Choices):
+    DOMAIN = "DOMAIN"
+    SLACK = "SLACK"
+
+
+class TrustedActivitiesClient:
+    """A client to expose the trusted activities/data preferences API
+
+    `Rest documentation <https://developer.code42.com/api/#tag/Trusted-Activities>`__
+    """
+
+    def __init__(self, trusted_activities_service):
+        self._trusted_activities_service = trusted_activities_service
+
+    def get_all(self, type=None, page_size=None):
+        """Gets all trusted activities.
+        `Rest documentation <https://developer.code42.com/>`
+
+        Args:
+            type (str, optional): Type of the trusted activity. `TrustedActivityType.DOMAIN` or `TrustedActivityType.SLACK`
+
+        Returns:
+            :class:'py42.response.Py42Response'
+        """
+        return self._trusted_activities_service.get_all(type, page_size)
+
+    def create(self, type, value, description=None):
+        """Gets all trusted activities with the given type.
+        `Rest documentation <https://developer.code42.com/>`
+
+        Args:
+            type (str): Type of the trusted activity. `TrustedActivityType.DOMAIN` or `TrustedActivityType.SLACK`
+            value (str): The URL of the domain or name of the Slack workspace.
+            description (str, optional): Description of the trusted activity.
+
+        Returns:
+            :class:'py42.response.Py42Response'
```

Comment:
the character after `class:` and at the end of this line should be a backtick (`), not a single quote.